controllers/talosconfig_controller.go

```diff
@@ -214,17 +255,31 @@ func (r *TalosConfigReconciler) Reconcile(req ctrl.Request) (_ ctrl.Result, rerr
 	}
 
 	// Packet acts a fool if you don't prepend #!talos to the userdata
```

Comment:
I guess this comment should be moved down close to `PacketMachine` now

Reply:
That whole if/else block below relates to PacketMachine, as we have to handle patching MachinePool and Machines separately.

Reply:
ah, sorry, missed that